src/assets/_project/_blocks/scss-general/_qg-variables.scss

```diff
@@ -351,7 +351,7 @@ $text-underline-offset: 5px;
     outline-color: $outline-color;
     outline-offset: 2px;
     border-radius: $border-radius;
-    box-shadow: none;
+    box-shadow: none !important;
```

Comment:
Is there a way that doesn't require the `!important` flag? It's kind of a blunt instrument...